tasks/main.yml

```diff
@@ -9,6 +9,20 @@
   changed_when: false
   always_run: true
 
+- name: install selinux dependencies when selinux is installed
+  yum: name="{{item}}" state=installed
+  with_items:
+    - policycoreutils-python
+    - checkpolicy
+  when: sestatus.rc == 0 and (ansible_os_family == 'RedHat' or ansible_os_family == 'Oracle Linux')
+
+- name: install selinux dependencies when selinux is installed
```

Comment:
Should we indicate that the one task is for Ubuntu and the other is for RedHat, may be easier for debugging?